src/Tokenizer/Tokens.php

```diff
@@ -572,6 +584,43 @@ public function getNonWhitespaceSibling(int $index, int $direction, ?string $whi
         }
     }
 
+    /**
```

Comment:
In general, for any code:

I struggle to understand the need for code without that code being used. For now, it looks only as a dead code
( and I would love to have automation to prevent any deadcode being in repo, as side note ;) )

I also fail to understand how common this will get used - is it's needed only for very limited usecase (and thus encapsulated into method of class, where will be the usage) or spread across widely (thus should be exposed via "main" Tokens).

For that, I would love to see actual usage of code, eg in the PR that is actually using it.
We could also have a PR adding the code (without usage), and other PR build on top, that shows how it will be used.